ppx

```diff
@@ -1,15 +0,0 @@
-#!/usr/bin/env sh
```

Comment:
I thought this script was used by the npm package to allow the consumer to run the appropriate binary, unless you changed something? I think we still need this.

Reply:
Oh, I now understand why this is existed!

Reply:
Fixed errors reported by shellcheck and committed a slightly improved one.